x/leverage/gov_handler_test.go

```diff
@@ -0,0 +1,60 @@
+package leverage_test
+
+import (
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramsproposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	"github.com/stretchr/testify/require"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	umeeapp "github.com/umee-network/umee/app"
+	"github.com/umee-network/umee/x/leverage"
+	"github.com/umee-network/umee/x/leverage/types"
+)
+
+func TestUpdateRegistryProposalHandler(t *testing.T) {
+	app := umeeapp.Setup(t, false, 1)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{
+		ChainID: fmt.Sprintf("test-chain-%s", tmrand.Str(4)),
+		Height:  1,
+	})
+
+	k := app.LeverageKeeper
+	h := leverage.NewUpdateRegistryProposalHandler(k)
+
+	t.Run("invalid proposal type", func(t *testing.T) {
+		require.Error(t, h(ctx, &paramsproposal.ParameterChangeProposal{}))
+	})
+
+	t.Run("valid proposal", func(t *testing.T) {
+		k.SetRegisteredToken(ctx, types.Token{BaseDenom: "uosmo"})
+		k.SetRegisteredToken(ctx, types.Token{BaseDenom: "uatom", BaseBorrowRate: sdk.MustNewDecFromStr("5.0")})
```

Comment:
```suggestion
		k.SetRegisteredToken(ctx, types.Token{BaseDenom: "uatom", BaseBorrowRate: sdk.MustNewDecFromStr("0.05")})
```
Note: Base borrow rate of 5% would eb expressed as 0.05